test/integration/util/util.go

```diff
@@ -81,6 +81,18 @@ func (m *MinishiftRunner) Start() {
 	m.RunCommand(fmt.Sprintf("start %s", m.CommandArgs))
 }
 
+func (m *MinishiftRunner) CDKSetup() {
+	m.RunCommand(fmt.Sprintf("setup-cdk --force --minishift-home %s", os.Getenv(constants.MiniShiftHomeEnv)))
+}
+
+func (m *MinishiftRunner) IsCDK() bool {
+	cmdOut, _, _ := m.RunCommand("-h")
+	if strings.Contains(cmdOut, "setup-cdk") {
```

Comment:
Seems this is caused by the fact that no Println is shown for code outside of the feature setup.